syntax/zeek.vim

```diff
@@ -11,6 +11,7 @@ syn keyword zeekTodo TODO XXX FIXME NOTE contained
 
 syn match zeekDirective /\v\@(DEBUG|DIR|FILENAME)/
 syn match zeekDirective /\v\@(deprecated)/
+syn match zeekDirective /\v\@pragma [^ ]+/ nextgroup=zeekDirectiveArg
```

Comment:
This doesn't highlight the argument to a `@pragma push`, e.g., `foo` would not be highlighted in `@pragma push foo`. I think that is probably fine for now.

Reply:
Eh, I've put more `\s`, `[:space:]` and `(` and `)` in it and I think it now does a better job.